lib/auth/tls_test.go

```diff
@@ -2560,3 +2560,16 @@ func (s *TLSSuite) TestEventsClusterConfig(c *check.C) {
 	c.Assert(err, check.IsNil)
 	suite.ExpectResource(c, w, 3*time.Second, clusterNameResource)
 }
+
+func (s *TLSSuite) TestAPIBackwardsCompatibilityLogic(c *check.C) {
```

Comment:
add a case where gRPC server returns a different kind of error (not `NotImplemented`)

Reply:
revised